src/sync/pull.rs

```diff
@@ -16,6 +16,8 @@ use std::collections::hash_map::HashMap;
 use std::default::Default;
 use std::fmt::Debug;
 
+const PULL_VERSION: u32 = 0;
```

Comment:
Can you put this next to the PullRequest definition, and maybe move the comment describing version 0 to it?